consensus/istanbul/types.go

```diff
@@ -31,6 +31,8 @@ type Proposal interface {
 	// Number retrieves the sequence number of this proposal.
 	Number() *big.Int
 
+	Header() *types.Header
```

Comment:
I may have missed this, but what's the purpose of this new field?  I don't recall seeing it used anywhere in this PR.